tests/preprocessing/test_trips.py

```diff
@@ -254,7 +254,7 @@ def test_accessor(self, example_trip_data):
         # generate tours using the accessor
         trips_acc, tours_acc = trips.as_trips.generate_tours()
 
-        pd.testing.assert_frame_equal(trips_expl, trips_acc)
+        pd.testing.assert_frame_equal(trips_expl, trips_acc, check_frame_type=False)
```

Comment:
Why do we need to add this line? The returns shall also be of the same datatype?

Reply:
It added it because `trips_expl` is still a `GeoDataFrame` as the direct call of `ti.preprocessing.trips.generate_tours` does not change the type, while `trips.as_trips` creates a `TripsGeoDataFrame` object. Thus the types are different.